app/bundles/CoreBundle/Helper/CookieHelper.php

```diff
@@ -98,6 +98,6 @@ public function setCookie($name, $value, $expire = 1800, $path = null, $domain =
      */
     public function deleteCookie($name, $path = null, $domain = null, $secure = null, $httponly = null)
     {
-        $this->setCookie($name, '', time() - 3600, $path, $domain, $secure, $httponly);
+        $this->setCookie($name, '', -86400, $path, $domain, $secure, $httponly);
```

Comment:
What's the reason for value -86400s (-24h)?

Reply:
If you mean that why `-24h` instead of `-1h`: I picked that number to counter all possible or impossible timezone difference

Reply:
Makes total sense. Thanks!